specification/managementgroups/resource-manager/Microsoft.Management/stable/2020-05-01/examples/GetAllSubscriptionsFromManagementGroup.json

```diff
@@ -0,0 +1,28 @@
+{
+  "parameters": {
+    "api-version": "2020-05-01",
+    "groupId": "Group"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "name": "728bcbe4-8d56-4510-86c2-4921b8beefbc",
+            "id": " /providers/Microsoft.Management/managementGroups/Group/subscriptions/728bcbe4-8d56-4510-86c2-4921b8beefbc",
+            "type": "Microsoft.Management/managementGroups/subscriptions",
+            "properties": {
+              "displayName": "Group",
+              "parent": {
+                "id": "/providers/Microsoft.Management/managementGroups/Group"
+              },
+              "state": "Active",
```

Comment:
>"state": "Active", [](start = 14, length = 18)

How do we keep these properties synced?

Reply:
Management Groups receives subscription notifications from ARM and Provisioning team if there are any changes to Subscription state